app/src/main/res/layout/rc_configuration_activity.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:local="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/ColorPrimary"
+        android:minHeight="?attr/actionBarSize"
+        local:popupTheme="@style/Theme.AppCompat.Light.NoActionBar"
+        local:theme="@style/ThemeOverlay.AppCompat.Dark" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/blue_300"
```

Comment:
Can we make the background white please?

Reply:
ok

Reply:
new design : 
![update_user_id](https://user-images.githubusercontent.com/10114329/132657958-1802c8f4-ad3b-4f02-8d1d-7c435ea2562e.jpg)

Reply:
Is there a "user id" value in the field? Or this is the label acting as a placeholder and when clicking on it it goes up to be the label? 

If it's the label, this is confusing. We should either remove it or make it light grey.

Is it also possible to have a grey border around the user id input?

Reply:
For your information, here's how it looks like on iOS, the spacing between the elements is also better on iOS IMO:

![241219001_169593998633688_7668251617385336400_n](https://user-images.githubusercontent.com/6832100/132663922-a92e3c7e-c37c-4afb-acc2-5e8fd8be1068.jpg)

Reply:
<img width="475" alt="Capture d’écran 2021-09-09 à 13 51 15" src="https://user-images.githubusercontent.com/10114329/132680851-b1548868-9bbb-4bb5-a47d-423f7aceacc8.png">
<img width="479" alt="Capture d’écran 2021-09-09 à 13 51 24" src="https://user-images.githubusercontent.com/10114329/132680864-cbc84a4b-656b-4d39-b3fe-88d2c9556e23.png">

Reply:
Ok, let's keep it like that for now, we'll improve it some more in the source configuration addition.